src/components/static/Navigation.svelte

```diff
@@ -108,12 +108,12 @@
           on:click={toggleMenu}
           aria-expanded="true"
           aria-controls="sidebar"
-          class="p-2 mr-2 text-white rounded-lg cursor-pointer lg:hidden hover:bg-mcswf-dark-blue hover:text-mcswf-gold"
+          class="text-white rounded-lg cursor-pointer lg:hidden hover:bg-mcswf-dark-blue hover:text-mcswf-gold"
         >
         {#if isMenuHidden}
         <Icon class="hover:text-mcswf-gold" icon="mdi:menu" />
         {:else}
-        <Icon class="hover:text-mcswf-gold" icon="mdi:close" />
+        <Icon height="20" width="20" class="hover:text-mcswf-gold" icon="mdi:close" />
```

Comment:
Please change this to `class="h-5 w-5"`

Please refer to https://tailwindcss.com/docs/height